### PR TITLE
made tests and populated index

### PR DESCRIPTION
**Pull request by amoralescr85**
Made a test that ensured that the index page was populating with shows.

### CODE REVIEW

**Comment by harrisonmuskat** [comment]
[As a user I want to view a list of items so that I can pick items to review](https://trello.com/c/37rK8pRg/25-as-a-user-i-want-to-view-a-list-of-items-so-that-i-can-pick-items-to-review)

File: spec/models/show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+feature "visitors can see a list of shows on homepage" do
+  scenario "visitor sees shows displayed" do
+
+    game_of_thrones = Show.create(title: 'Game of Thrones', description: 'awesome HBO show', average_rating: 5)
+    californication = Show.create(title: 'Californication', description: 'awesome Showtime show', average_rating: 5)
```
Comment (justinhuynh) [inline spec/models/show_spec.rb]: This line on the longer side...consider splitting it up like so:

```
game_of_thrones = Show.create(
  title: 'Game of Thrones',
  description: 'asdfasdfasdfadsf'
...
)
```

File: app/models/show.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Show < ApplicationRecord
+
```
Comment (justinhuynh) [inline app/models/show.rb]: Ditch the extra blank lines

File: react/src/containers/App.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+
+import Show from '../components/Show'
+
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      shows
```
Comment (justinhuynh) [inline react/src/containers/App.js]: Are you trying to assign `shows` to a null value? Or is `shows` coming in from somewhere? Just be sure to initialize your variable properly

File: react/src/containers/App.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  }
+
+  render() {
+    return(
+      <div>
+        <div className="row">
+          <div className="small-12 columns">
+            <h1>Must Watch TV Shows</h1>
```
Comment (justinhuynh) [inline react/src/containers/App.js]: We may need to update your `.hound.yml` files to fix this

**Pull request merged**